spec/integration/resource/api200/login_detail/create.rb

```diff
@@ -0,0 +1,9 @@
+require 'spec_helper'
+
+klass = OneviewSDK::LoginDetail
+RSpec.describe klass, integration: true, type: CREATE, sequence: seq(klass) do
+  let(:current_client) { $client }
+  let(:login_detail_class) { OneviewSDK::LoginDetail }
```

Comment:
The variable `login_detail_class` is not need because the shared example `LoginDetailGetExample` is not using it. So this line can be removed.